.python-version

```diff
@@ -0,0 +1 @@
+3.10.4
```

Comment:
Including this isn't ideal.  It forces pyenv users to have that specific bugfix version installed versus eg `3.10`.  We've generally tended to only use it for projects which require a non-latest major Python version to avoid the fallout of it.  We also include a fallback in the justfile so the default venv setup can pick a minor version.

Reply:
Very odd: `just devenv` was complaining about requirements without hashes, which happens when using an incorrect Python version. Hence, adding _.python-version_. But I've removed _.python-version_ and `just devenv` isn't complaining any more. (I have 3.10 available globally.) Possibly I had an old, incorrect virtual environment?

I'll remove this commit and force push.

Reply:
Possibly just should run `clean` before `virtualenv`?